spec/lib/secure_headers/configuration_spec.rb

```diff
@@ -29,16 +29,14 @@ module SecureHeaders
       expect_default_values(header_hash)
     end
 
-    it "copies all config values except for the cached headers when dup" do
+    it "copies config values when duping" do
```

Comment:
Added a test [here](https://github.com/twitter/secureheaders/pull/232/files#diff-5ffee118dcfe384935df780237dadf39R109) for the cache busting. I'll have to think about your second item.